pkg/api/util/db_test.go

```diff
@@ -249,29 +249,85 @@ func TestCreateVolumeAttachmentDBEntry(t *testing.T) {
 	var req = &model.VolumeAttachmentSpec{
 		BaseModel: &model.BaseModel{},
 		VolumeId:  "bd5b12a8-a101-11e7-941e-d77981b584d8",
-		Metadata:  map[string]string{"a": "a"},
 		Status:    "creating",
 	}
-	var vol = &model.VolumeSpec{
+
+	// Test case 1: Volume status should be available that attachment can be created.
```

Comment:
Please use `t.Run()` to split them into sub-testcases.

Reply:
Done